//Test Script

/*
integer age1 = 40;
integer age2 = 80;
integer agesum = age1 + age2;
system.debug(agesum);

*/
PLACEHOLDER
public class CustomException extends Exception{}
try{
  system.assertEquals(120,agesum);
} catch(Exception exc){
    throw new CustomException('there is a error '+exc.getMessage());
}
